### ISSUE TITLE
An extra .flo/.pfm file

### ISSUE THREAD

**Issue by eli5168** [lmb-freiburg/flownet2#79]
Now I just wanna produce optical flow, and I understand two consecutive frames get one .flo/.pfm file. That means ten consecutive images produce nine .flo/.pfm files. However, there are still ten .flo/.pfm files existing at opticl flow left/right directory. So, why is there an extra .flo/.pfm file? What can it be used for? Thanks.

**Comment by nikolausmayer** [comment]
You're right — if "classical" optical flow between two frames is all you want, then there's no use for that extra file and you can ignore it.

One example for a task which can make use of it would be "next flow prediction" (explored in e.g. https://arxiv.org/abs/1612.03777)
